pom.xml

```diff
@@ -444,6 +416,12 @@
                     <container>
                         <expandClasspathDependencies>true</expandClasspathDependencies>
                     </container>
+                    <from>
+                        <image>416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-corretto-build-21:latest</image>
+                    </from>
+                    <to>
+                        <image>416670754337.dkr.ecr.eu-west-2.amazonaws.com/local/psc-statement-delta-consumer:latest</image>
```

Comment:
same question as before for this image

Reply:
Just checking with Abi

Reply:
Using old image reference until staging deployment done